### PR TITLE
points track character

### PR DESCRIPTION
**Pull request by ThomasGITH**
Dit is de character met points om te vergelijken.

### CODE REVIEW

File: FitTheWall/Assets/PositionTo.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PositionTo : MonoBehaviour
+{
+    public GameObject objectToTrack;
```
Comment (Spraxs) [inline FitTheWall/Assets/PositionTo.cs]: comment variable.

Also, does this really need to be public? Let me know.

I bet we can make it a "[SerializeField] private"

File: FitTheWall/Assets/PositionTo.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    void Update()
+    {
+        gameObject.transform.position = new Vector3(objectToTrack.transform.position.x, objectToTrack.transform.position.y, gameObject.transform.position.z);
```
Comment (Spraxs) [inline FitTheWall/Assets/PositionTo.cs:23]: Create a function for this action & comment this function. What does it do?

File: FitTheWall/Assets/SilhouetteManager.cs [hunk 3]
```diff
@@ -0,0 +1,21 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SilhouetteManager : MonoBehaviour
+{
+
+    //public List silhouettes = new List<GameObject>();
```
Comment (Spraxs) [inline FitTheWall/Assets/SilhouetteManager.cs]: Remove commented out code

**Review by Spraxs — CHANGES_REQUESTED** [review]

File: FitTheWall/Assets/PositionTo.cs (hunk 1 above)
Comment (ThomasGITH) [reply FitTheWall/Assets/PositionTo.cs]: @Spraxs It's public in order to be able to drag in objects from inside the Unity editor.

Comment (Spraxs) [reply FitTheWall/Assets/PositionTo.cs]: Try [SerializeField] private

File: FitTheWall/Assets/PositionTo.cs [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+public class PositionTo : MonoBehaviour
+{
+    [SerializeField] private GameObject objectToTrack;
+
+    void Update()
+    {
+        AssignXYPosition();
+    }
+
+    void AssignXYPosition()
```
Comment (Spraxs) [inline FitTheWall/Assets/PositionTo.cs:21]: Comment function

File: FitTheWall/Assets/PositionTo.cs [hunk 5]
```diff
@@ -0,0 +1,21 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+//used to assign the two-dimensional (X and Y) coordinates to this object's transform
```
Comment (Spraxs) [inline FitTheWall/Assets/PositionTo.cs]: Use summary comments for classes & functions

File: FitTheWall/Assets/PositionTo.cs [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+public class PositionTo : MonoBehaviour
+{
+    [SerializeField] private GameObject objectToTrack;
```
Comment (Spraxs) [inline FitTheWall/Assets/PositionTo.cs:11]: Comment variable

**Review by Spraxs — CHANGES_REQUESTED** [review]

File: FitTheWall/Assets/PositionTo.cs (hunk 4 above)
Comment (Spraxs) [reply FitTheWall/Assets/PositionTo.cs:21]: use summary format

File: FitTheWall/Assets/PositionTo.cs (hunk 6 above)
Comment (Spraxs) [reply FitTheWall/Assets/PositionTo.cs:11]: Use the "//" format

File: FitTheWall/Assets/PositionTo.cs [hunk 7]
```diff
@@ -0,0 +1,21 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+/// used to assign the two-dimensional (X and Y) coordinates to this object's transform
```
Comment (Spraxs) [inline FitTheWall/Assets/PositionTo.cs:6]: use summary format

File: FitTheWall/Assets/PositionTo.cs [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+{
+    //The object which coordinates are taken
+    [SerializeField] private GameObject objectToTrack;
+
+    void Update()
+    {
+        AssignXYPosition();
+    }
+
+    /// Replaces X and Y coordinates with that of 'objectToTrack'
```
Comment (Spraxs) [inline FitTheWall/Assets/PositionTo.cs]: use summary format

**Review by Spraxs — CHANGES_REQUESTED** [review]

File: FitTheWall/Assets/PositionTo.cs (hunk 7 above)
Comment (Spraxs) [reply FitTheWall/Assets/PositionTo.cs:6]: https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/

File: FitTheWall/Assets/PositionTo.cs (hunk 8 above)
Comment (Spraxs) [reply FitTheWall/Assets/PositionTo.cs]: https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/

**Pull request merged**